adapters/repos/db/vector/hnsw/distancer/asm/dot_avx256_amd64.go

```diff
@@ -1,16 +1,4 @@
-//                           _       _
```

Comment:
These should not be modified, you probably ran `go generate` or goat on the whole package ?

You should revert these changes and only provide a PR with the necessary changes 😉